Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/wrapperUtils/Common.java

```diff
@@ -118,22 +118,29 @@ public static <C extends ComplexType<C>> ImgPlus<BitType> toBitTypeImgPlus(
 
 	/**
 	 * Shows a warning dialog about image anisotropy, and asks if the user wants
-	 * to continue.
+	 * to continue. If the 
```

Comment:
Comment sentence needs completing? sth like:
```suggestion
	 * to continue. If the plugin is running in headless mode, execution continues anyway, and a warning is printed to the log.
```